config.base.yaml

```diff
@@ -28,6 +38,9 @@ audits:
       - mockevent
     alerts:
       - filestore
+      - emailstore
+
+
```

Comment:
Please remove the additional blank line here.

Reply:
Done